renderer/skus/js_skus_browsertest.cc

```diff
@@ -43,8 +41,12 @@ TEST_F(JsSkusBrowserTest, AttachSkus) {
   GURL url("https://account.brave.software");
   LoadHTMLWithUrlOverride(R"(<html><body> </body></html>)", url.spec().c_str());
   EXPECT_TRUE(ExecuteJavascript(command));
-  Reload(url);
-  EXPECT_TRUE(ExecuteJavascript(command));
+  // Reloading the url doesn't seem to work any more presumably because of the
+  // change https://source.chromium.org/chromium/chromium/src/+/
+  // ae845bfbaace3a356b66de078d6d70c84192c7f7, which causes an empty
+  // security origin and skus::IsSafeOrigin then obviously returns false.
+  // Reload(url);
```

Comment:
I have no idea what he was testing here, we should follow-up to look at this test @bsclifton

Reply:
@bsclifton and I discussed it before and he thought it was safe to just drop the reload test. I will delete these lines.